src/util.ts

```diff
@@ -48,3 +49,10 @@ export const validateSignature = async (
     throw Error(`Signature verification failed for file path: ${filePath}`);
   }
 };
+
+export const constructURL = (base: string, path: string): string =>
```

Comment:
[suggestion] maybe we can rename this to `constructURLHref` since we are returning the href rather than URL object